### ISSUE TITLE
Thumbnails/Images not showing in lighttable after import (but does work in Adobe Bridge)

### ISSUE THREAD

**Issue by stevecs** [darktable-org/darktable#15221]
### Describe the bug

I have thousands of scanned images (slides/negatives/positives) captured with VueScan in (64/16bit) RGBI DNG format on a network share.  (Images scanned with Nikon LS-9000 and Epson Perfection V850).  I have historically been using Adobe Bridge to manage photos and wanted to see about converting over to Darktable/Lighttable).  When importing a copy of the directory (as a test to avoid any bugs with image modification) lighttable imports the files, but all show up with a missing image placeholder.  

During import I get a flash of 'image xxxxxxx.dng is not available' which is the filename of the actual image file.   The same file can be read natively and can be imported/seen in bridge without an issue.

After the import process the exif data on the file does exist (can see the metadata, path, filename, size, etc) but just no thumbnail and when I go to the darkroom tab it says 'could not load xxxxxx.dng switching to lighttable now'.  

But can go into photoshop from bridge and works fine there.

If I use individual JPEG images this works as expected.    Can Darktable not handle 64bit/16bit RGBI TIFF/DNG files or RAW (gamma 1.0) DNG files?

### Steps to reproduce

Import
Add to library.   (any directory),
select files to import.

### Expected behavior

should generate or display thumbnails of images in lighttable and allow processing with darkroom

### Where did you install darktable from?

darktable.org

### darktable version

4.4.2

### What OS are you using?

Windows

### What is the version of your OS?

Windows Server 2022 (21H2)

### Describe your system?

Supermicro X10DAX
Dual E5-2667V4
256GB ECC Ram
Windows server 2022 (21h2)
Nvidia GTX 3080TI

### Are you using OpenCL GPU in darktable?

None

**Comment by wpferguson** [comment]
Could you supply an image that we could test with?

**Comment by stevecs** [comment]
uploaded a sample here, it's 7zipped to save some space and to allow checking for transmission errors. 
https://gofile.io/d/35izkZ

This is one scanned with a nikon coolscan LS-9000 w/ vuescan software.

**Comment by jenshannoschwalm** [comment]
rawspeed does currently not support such images 

@LebedevRI a quick test, in `UncompressedDecompressor.cpp` we have in line 118
```
  if (cpp < 1 || cpp > 3)
    ThrowRDE("Unsupported number of components per pixel: %u", cpp);

```

changging to `if (cpp < 1 || cpp > 3)` seems to be fine - not understanding the rawspeed code though :-)

**Comment by LebedevRI** [comment]
> 

Can you please contribute such a sample to RPU?
But please upload something with a small file size, 1MPix should be plenty here.

**Comment by LebedevRI** [comment]
Once again, can you not specify the target size in the software?
181MiB is a bit too much for this.

**Comment by ralfbrown** [comment]
Or scan at a lower resolution.

**Comment by jenshannoschwalm** [comment]
And please update the issue title, this issue is very specific. It's about 4channel dng files with the forth channel being infrared.

**Comment by stevecs** [comment]
I don't have the nikon LS9000 attached at the moment, but have the following (same issue) scanned from the Epson PerfectionV850 scanner.

https://gofile.io/d/5GYJZG

also uploaded to RPU.   Much smaller scan and using a different scanner (but same capture software) to aid in fixing issue and hopefully showing it to be scanner agnostic (i.e. related to VueScan RGBI/4channel opposed to a particular scanner model).

**Comment by LebedevRI** [comment]
Unfortunately, that image contains people, so it's not well-suited for RPU...
Do you perhaps have another similar problematic sample? The filesize is nice!

**Comment by stevecs** [comment]
that's my family member, I will see about another (working through a current batch but they are all portraits).  I guess I am confused as to the purpose of RPU here, I mean the image itself should be irrelevant is it not?   The point really is the 4 channel nature of the TIFF/DNG capture and handling that within Darktable.

**Comment by LebedevRI** [comment]
Thank you!

> I mean the image itself should be irrelevant is it not?

Sure, but that is a publicly-accessible repository, and the upload is under a CC0 license.
I can't imagine that is something you'd want to generally do with a picture of a person.

**Comment by stevecs** [comment]
True, though in this particular case it's a sad fact that the person is deceased.    Do you still need me to find another photo or will the ones provided suffice?

**Comment by LebedevRI** [comment]
As per the rules (suggestions?) noted on https://raw.pixls.us/:
> We are NOT looking for:
> Photographs of people, for legal reasons.

yes.

**Comment by vsz1lu** [comment]
I can't reproduce on Linux. Ubuntu 22.04; dt 4.4.2; Epson V550; C-41 neg;  VueScan 9.7.30 RGBI tiff (scanned 6 years ago). See the video.
[Képernyőfelvétel ekkor: 2023-09-14 16:42:27.webm](https://github.com/darktable-org/darktable/assets/57588193/75eb91b3-a343-4ddb-9815-0eefe05fa58d)

**Comment by LebedevRI** [comment]
@stevecs ping

**Comment by suntorytimed** [comment]
Do you still require sample files for this bug? Unfortunately I ran into this issue when I wanted to use darktable to develop my negative scans. 48bit RAW DNG without infrared channel don't work as well. I scanned with a Nikon 9000ED and VueScan.
